Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -570,33 +570,57 @@ + (MAAdapterError *)toMaxError:(nullable NSError *)vungleError
             break;
         case 2: // Invalid AppID
         case 201: // Invalid PlacementID
+        case 207: // Invalid Placement Type
+        case 222: // Invalid Placement load type
         case 500: // BannerView: Invalid Size
             adapterError = MAAdapterError.invalidConfiguration;
             break;
+        case 202: // Ad already Consumed
+        case 203: // Ad is already loading
+        case 204: // Ad already loaded
+        case 205: // Ad is playing
+        case 206: // Ad already failed loading
         case 210: // Ad Not Loaded
-            adapterError = MAAdapterError.noFill;
+            adapterError = MAAdapterError.invalidLoadState;
```

Comment:
instead of handling the `isPlayFlow` check outside the switch, why not assign the adapter error using a ternary here?

`adapterError = isPlayFlow ? MAAdapterError.adNotReady : MAAdapterError.invalidLoadState;`